gapic/samplegen_utils/snippet_metadata_pb2.py

```diff
@@ -1,12 +1,10 @@
 # -*- coding: utf-8 -*-
 # Generated by the protocol buffer compiler.  DO NOT EDIT!
-# source: snippet_metadata.proto
-
-# type: ignore
```

Comment:
Please restore this line to allow mypy to pass

Reply:
Done